data/json/effects.json

```diff
@@ -676,6 +676,60 @@
             "sleep_chance": [101]
         }
     },
+    {
+        "type": "effect_type",
+        "id": "bandaged",
+        "name": ["Bandaged"],
+        "desc": ["Your wounds are bandaged."],
+        "main_parts_only": true,
+        "rating": "good",
+        "int_dur_factor": 3600,
+        "max_intensity": 16,
+        "//": "How many intensity levels may apply effects",
+        "max_effective_intensity": 8,
+        "max_duration": 57600,
+        "base_mods": {
+            "//": "Default healing rate (x100000)", 
+            "healing_rate": [20],
```

Comment:
That `x100000` part is non-standard. Additionally, other multipliers below use different rates.
Choose standard rates to avoid confusing contributors with multiple different rate that need to be looked up.

For absolute effects, pick one of:
* Per turn
* Per day
* Over whole expected duration

For multipliers, pick one of:
* Percentage (`* 0.01`)
* `float` multiplier - ie. 1 is default, 0.5 is half etc.